buyer-api/src/queues/registerPaymentsQueue.js

```diff
@@ -2,12 +2,15 @@ import { createQueue } from './createQueue';
 import { registerPayment } from '../operations/registerPayment';
 import logger from '../utils/logger';
 
+const queue = createQueue('RegisterPayments');
+
+const pauseQueue = async () => queue.pause();
+
 const process = async ({ id, data: { notarizationRequestId } }) => {
   logger.info(`NR[${id}] :: Process :: ${notarizationRequestId}`);
-  return registerPayment(notarizationRequestId);
+  return registerPayment(notarizationRequestId, pauseQueue, id);
```

Comment:
I think this a really good approach to stop registering payments, let's to push it forward so the business knows little about it. What if:
```js
  return registerPayment(notarizationRequestId, (reason) => {
    logger.info(`NR[${id}] :: RegisterPayments queue paused :: ${reason}`);
    return queue.pause();
  });
```
So, inside `registerPayment` you should do:
```js
if (...condition not met...) {
  await pauseQueue('Not enough account balance ...')
}
```